<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseCover_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать обложку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Если обложка помещается в рамки ниже, то можете ее использовать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBook_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>690</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить книгу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Название книги</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseText_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>630</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать txt файл с текстом книги</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ID автора</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
